docs/PublishingGuide.md

```diff
@@ -22,6 +24,22 @@
 It takes a few minutes for report generation. Meanwhile, you can monitor the progress live at [Travis-CI's Builds](https://travis-ci.org/dashboard/builds). <br/>
 Try accessing your site again when a green tick appears beside your fork.
 
+### Using GitHub Actions
+
+1. Fork publish-RepoSense repository using this [link](https://github.com/RepoSense/publish-RepoSense/fork)
+1. Follow this [section](#granting-write-access-to-reposense-for-publishing) to generate a `personal access token` or `deploy key` on GitHub for report publishing 
+1. Go to the [secrets settings](../../../../publish-RepoSense/settings/secrets) of your publish-RepoSense fork, add a new secret as `ACCESS_TOKEN` or `DEPLOY_KEY` depending on your earlier choice and paste the content that was copied earlier to its value field; then click **Add secret** <br/>
+![GitHub Actions Secrets](images/publishingguide-secrets.png "GitHub Actions Secrets")
+1. Edit [run.sh](../../../../publish-RepoSense/edit/master/run.sh), [repo-config.csv](../../../../publish-RepoSense/edit/master/configs/repo-config.csv) and [author-config.csv](../../../../publish-RepoSense/edit/master/configs/author-config.csv) to customize the command line parameters or repositories to be analyzed <br/>
+*Read our [User Guide](UserGuide.md#customizing-the-analysis) for more information*
```

Comment:
Add this line in a different manner. The rendered view file doesn't look good. Maybe can just add it to the previous line or use `>`.

Reply:
This line was copied from `Using Travis` section. I did not make changes to it for consistency.

Reply:
Yes noticed that. Can you just edit in both the places? Looks a bit weird currently.

![Screen Shot 2020-03-08 at 10 31 10 PM](https://user-images.githubusercontent.com/35946746/76164815-dbd97a00-618c-11ea-9e6c-7177c98aeb82.png)

> Could you highlight what punctuation marks you are referring to?

Add full stops at the end of the steps and sentences. You can do this for both the sections too.